src/i18n/dim_de.json

```diff
@@ -1,481 +1,491 @@
-         "GitHub": "GitHub",
-         "GitHubHelp": "Wenn du an dem Projekt mitwirken möchtest, besuche uns auf unserer Projektseite auf GitHub.",
-         "GitHubRepo": "DIM Git Repo",
-         "Header": "Über Destiny Item Manager (DIM)",
-         "HowItsMade": "DIM basiert auf denselben Diensten wie Bungie.net und der Destiny Companion App. DIM kann auf die Items in deinem Hüter-Inventar und dem Tresor auf PlayStation und Xbox zugreifen. Du kannst dann Elemente per Drag & Drop verschieben, wenn Du sie verschieben möchtest.",
-         "Reddit": "Reddit",
-         "RedditHelp": "Wenn du Fragen, Kommentare oder Anregungen hast - lass es uns wissen!",
-         "Twitter": "Twitter",
-         "TwitterHelp": "Folge uns, um Informationen zu DIM erhalten und immer auf dem neuesten Stand zu sein.",
-         "Version": "Version {version} ({flavor}), erstellt am {date}",
-         "Transifex": "Trete dem Übersetzer-Team bei!",
-         "TransifexText": "Wir benutzen Transifex für eine unkomplizierte Übersetzung. Wenn du die DIM Übersetzung in einer Sprache verbessern möchtest, trete dem Team bei."
-      },
-      "Support": {
-         "Support": "Unterstütze DIM",
-         "FreeToDownload": "DIM ist ein kostenloses Tool. Der Quellcode ist Open-Source und kann von jedem verbessert werden. Du wirst niemals Werbung in DIM sehen. Das ist unser Versprechen.",
-         "OpenCollective": "Wir benutzen den OpenCollective Service um die Hingabe und Zeit zu kompensieren, die unsere Entwickler in dieses Projekt gesteckt haben.",
-         "Backers": "Unterstützer",
-         "BackersDetail": "Unterstütze uns mit einer einmaligen oder monatlichen Spende, um uns zu helfen aktiv weiterzuentwickeln",
-         "Sponsors": "Sponsoren",
-         "SponsorsDetail": "Werde Sponsor und platziere hier dein Logo mit einem Link zu deiner Seite.",
-         "Teespring": "Wir haben auch T-Shirts mit unserem neuen Logo zum kaufen auf Teespring:",
-         "TeespringLink": "DIM bei Teespring"
-      },
-      "UpgradeChrome": "Deine Chrome-Version ist zu alt für DIM. Bitte aktualisiere Chrome um DIM wieder nutzen zu können.",
-      "Login": {
-         "Permission": "Wir brauchen dein Einverständnis....",
-         "Explanation": "Erlaube DIM deinen Destiny Charakter,  Tresor und Fortschritt einzusehen und zu bearbeiten.",
-         "Auth": "Autorisiere mit Bungie.net",
-         "LearnMore": "Willst du mehr wissen?"
-      }
-   },
-   "TrialsCard": {
-      "FiveWins": "5 Siege Belohnung (Rüstung)",
-      "SevenWins": "7 Siege Belohnung (Waffe)",
-      "Flawless": "Makellos"
-   }
+    "Bucket": {
+        "Armor": "Rüstung",
```

Comment:
The download service for the other languages will always output with these no matter how I upload them. So English will be normal and the others will have consistent spacing amongst them.